### PR TITLE
Setup repo to publish nugets through github actions.

### PR DESCRIPTION
**Pull request by atmgrifter00**

### CODE REVIEW

File: .github/workflows/dotnet-core.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      with:
+        dotnet-version: 3.1.301
+    - name: Install dependencies
+      run: dotnet restore
+    - name: Build
+      run: dotnet build --configuration Release --no-restore
+    - name: Publish WindowsAPICodePack
+      uses: brandedoutcast/publish-nuget@v2.5.2
+      with:
+          PROJECT_FILE_PATH: ../../../source/WindowsAPICodePack/Core/Core.csproj
```
Comment (atmgrifter00) [inline .github/workflows/dotnet-core.yml:25]: I believe this is the right relative path to the project file. Someone please check me on this.

File: .github/workflows/dotnet-core.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name: .NET Core
```
Comment (alejandro5042) [inline .github/workflows/dotnet-core.yml:1]: Please name this pipeline something like:

```
Build and Publish (.NET Core)
```

File: .github/workflows/dotnet-core.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.301
+    - name: Install dependencies
+      run: dotnet restore
+    - name: Build
```
Comment (atmgrifter00) [inline .github/workflows/dotnet-core.yml:20]: @alejandro5042, do you know how this knows to build the solution file (located in the 'source/WindowsAPICodePack' folder?

File: .github/workflows/dotnet-core.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      run: dotnet build --configuration Release --no-restore
+    - name: Publish WindowsAPICodePack
+      uses: brandedoutcast/publish-nuget@v2.5.2
+      with:
+          PROJECT_FILE_PATH: ../../../source/WindowsAPICodePack/Core/Core.csproj
+          NUGET_KEY: ${{secrets.NUGET_API_KEY}}
```
Comment (atmgrifter00) [inline .github/workflows/dotnet-core.yml:26]: @alejandro5042 , the NUGET_API_KEY is still something we have to setup first correct?

Comment (alejandro5042) [reply .github/workflows/dotnet-core.yml:26]: Yep! See: https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets

Also talk to @benmont about getting a nuget.org API key for our org.

File: .github/workflows/dotnet-core.yml (hunk 3 above)
Comment (alejandro5042) [reply .github/workflows/dotnet-core.yml:20]: Depends:

> If a project or solution file isn't specified, MSBuild searches the current working directory for a file that has a file extension that ends in either proj or sln and uses that file.

See: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build

Comment (atmgrifter00) [reply .github/workflows/dotnet-core.yml:20]: Updated with a path to the solution file.

**Pull request merged**